.github/workflows/npm-release-publish.yml

```diff
@@ -26,13 +26,6 @@ jobs:
         with:
           node-version: '14.x'
 
-      # Try get node_modules from cache
```

Comment:
This not required anymore?

Reply:
Forgot to mention that in the PR desc, I think we should not be using cached node modules for release and instead making sure the install is 100% fresh

Reply:
It shouldn't matter and we should be able to use the cached node modules - I'm just being extra paranoid -- happy to put this back in or keep out